static/install

```diff
@@ -101,6 +98,11 @@ curl --fail --location --progress-bar --output "$tmp_tar" "$dune_tar_uri" ||
 tar -xf "$tmp_tar" -C "$tmp_dir" > /dev/null 2>&1 ||
     error "Failed to extract dune archive content from \"$tmp_tar\""
 
+# locate the name of the directory that was unpacked in the tarball
+tmp_tar_dir="$(find "$tmp_dir" -mindepth 1 -maxdepth 1 -type d)"
```

Comment:
```suggestion
tmp_tar_dir="$(find "$tmp_dir" -mindepth 1 -maxdepth 1 -name "dune-*" -type d)"
```
It's me being neat picky by adding another layer of verification to make sure we don't find another kind of repository. Feel free to ignore if you don't consider it important.

Reply:
I think I prefer https://github.com/ocaml-dune/binary-distribution/pull/108 which moves the temporary directory to a newly created, non-hardcoded path. That way we don't add too many arbitrary constraints on the tarball while avoiding accidental capture of potential other files in the temporary directory.